crypto/pem/pem_info.c

```diff
@@ -292,9 +292,9 @@ int PEM_X509_INFO_write_bio(BIO *bp, X509_INFO *xi, EVP_CIPHER *enc,
             }
 
             /* create the right magic header stuff */
-            OPENSSL_assert(strlen(objstr) + 23
-                           + 2 * EVP_CIPHER_iv_length(enc) + 13 <=
-                           sizeof buf);
+            if (!ossl_assert(strlen(objstr) + 23 + 2 * EVP_CIPHER_iv_length(enc)
+                             + 13 <= sizeof buf))
```

Comment:
And if you choose to do something about it, I'd like to suggest even parenthesis around multi-line expression.